presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -0,0 +1,37 @@
+datasource: presto
+query-names: presto/tpch/${query}.sql
+runs: 6
```

Comment:
template for temporary copy of benchmark until we implement: https://github.com/prestodb/benchto/issues/37. 
Another option is to wait for https://github.com/prestodb/benchto/issues/37 and then merge `presto-benchmarks` module. 
Depends how fast you need benchmarks module to land in.